docs/docsite/rst/installation_guide/intro_installation.rst

```diff
@@ -143,18 +101,24 @@ You may need to perform some additional configuration before you are able to run
 Installing Ansible
 ------------------
 
-Use ``pip`` in your selected Python environment to install the Ansible package of your choice for the current user:
+Use ``pip`` in your selected Python environment to install the minimal ``ansible-core`` package for the current user:
```

Comment:
Changing the order here changes what this heading is.  The original had Ansible package first as that's the heading "Installing Ansible".  This was a conscious decision to make Ansible the package the prominent and preferred installation.

If community folks feel differently, then this needs to be two separate sections now with a heading for core and a heading for package installs.

Reply:
Changed the order as it was